alacritty/src/migrate.rs

```diff
@@ -172,6 +172,13 @@ fn migrate_renames(config: &mut Value) -> Result<(), String> {
     // mouse_bindings -> mouse.bindings
     move_value(config_table, &["mouse_bindings"], &["mouse", "bindings"])?;
 
+    // Avoid warnings due to introduction of the new `general` section.
+    move_value(config_table, &["live_config_reload"], &["general", "live_config_reload"])?;
+    move_value(config_table, &["working_directory"], &["general", "working_directory"])?;
+    move_value(config_table, &["ipc_socket"], &["general", "ipc_socket"])?;
+    move_value(config_table, &["import"], &["general", "import"])?;
+    move_value(config_table, &["shell"], &["terminal", "shell"])?;
```

Comment:
Could we have a way to use `alacritty migrate` for `toml` to `toml` transition to reduce user amount of work? I think it should be relatively trivial and we already discussed something like that in the past?

Though, I won't block this PR on this small thing, just would be nice for the next release and other future releases.

Reply:
> Could we have a way to use alacritty migrate for toml to toml transition to reduce user amount of work? I think it should be relatively trivial and we already discussed something like that in the past?

I have considered doing this but decided against it because rewriting the toml config is going to mess up formatting, things like keybindings, nuke comments. I think to do that in a reasonable manner we'd have to preserve that stuff and I don't know how.

Do you think this is feasible?

Reply:
I'm pretty sure you can do that with https://docs.rs/toml_edit/latest/toml_edit/ which we already use.

it explicitly states that everything is _preserved_ and just added and that's how `cargo add` works, which also adds things to toml, but preserves everything inside of it.

Reply:
Just thinking about it, your issue is the comment above the key we're about to move? Maybe https://docs.rs/toml_edit/latest/toml_edit/struct.Formatted.html kind of captures it?

Reply:
I'll investigate it. Might as well do so with this PR.